src/panels/OriginPanel/OriginPanel.tsx

```diff
@@ -192,32 +193,40 @@ export function OriginPanel() {
             </Title>
             <OriginSettings />
           </Group>
-          <Group gap={0} mb={'xs'}>
-            <ActionIcon
-              onClick={() => setNavigationAction(NavigationActionState.Focus)}
-              aria-label={'Select focus'}
-              size={'lg'}
-              style={actionIconStyle(NavigationActionState.Focus)}
-            >
-              <FocusIcon size={'70%'} />
-            </ActionIcon>
-            <ActionIcon
-              onClick={() => setNavigationAction(NavigationActionState.Anchor)}
-              aria-label={'Select anchor'}
-              size={'lg'}
-              style={actionIconStyle(NavigationActionState.Anchor)}
-            >
-              <AnchorIcon size={'70%'} />
-            </ActionIcon>
-            <ActionIcon
-              onClick={() => setNavigationAction(NavigationActionState.Aim)}
-              aria-label={'Select aim'}
-              size={'lg'}
-              style={actionIconStyle(NavigationActionState.Aim)}
-            >
-              <TelescopeIcon size={'70%'} />
-            </ActionIcon>
-          </Group>
+          <SegmentedControl
+            value={navigationAction}
+            withItemsBorders={false}
+            onChange={(v) => setNavigationAction(v as NavigationActionState)}
```

Comment:
I think we should avoid single character variable names unless they are for indices

Reply:
So: 
```suggestion
            onChange={(state) => setNavigationAction(state as NavigationActionState)}
```